lib/routes/account.js

```diff
@@ -599,6 +599,97 @@ module.exports = function (
         }
       }
     },
+
+    {
+      method: 'POST',
+      path: '/account/login/send_authorization_code',
+      config: {
+        validate: {
+          payload: {
+            email: validators.email().required(),
+            service: isA.string().max(16).alphanum().optional(),
+            redirectTo: isA.string().uri().optional(),
+            resume: isA.string().optional()
+          }
+        },
+        response: {
+          schema: {
+          }
+        }
+      },
+      handler: function (request, reply) {
+        log.begin('Account.sendAuthorizationCode', request)
+
+        var authorizationCode = crypto.randomBytes(16)
+        var form = request.payload
+        var email = form.email
+        var service = request.payload.service || request.query.service
+        var redirectTo = request.payload.redirectTo
+        var resume = request.payload.resume
+        var emailRecord
+        var ip = request.app.clientAddress
+
+        return db.emailRecord(email)
+          .then(
+            function (account) {
+              return mailer.sendAuthorizeLoginEmail(
+                account,
+                authorizationCode,
+                {
+                  acceptLanguage: request.app.acceptLanguage,
+                  redirectTo: redirectTo,
+                  resume: resume,
+                  service: service
+                }
+              )
+            }
+          )
+          .then(
+            function () {
+              reply({})
+            },
+            reply
+          )
+      }
+    },
+    {
+      method: 'POST',
+      path: '/account/login/reject_authorization_code',
```

Comment:
I'm guessing this is a route that the content-server will call if a user sees an email, decides it wasn't them, and clicks "This wasn't me"?